### PR TITLE
Add active health check feature

### PR DESCRIPTION
**Pull request by MarianConstantinMarica**

### CODE REVIEW

File: spring-petclinic-vets-service/src/main/resources/application.yml [hunk 1]
```diff
@@ -8,6 +8,13 @@ spring:
   profiles:
     active: production
 
+management:
```
Comment (arey) [inline spring-petclinic-vets-service/src/main/resources/application.yml:11]: **question**: in the https://github.com/spring-petclinic/spring-petclinic-microservices-config/blob/main/application.yml#L32 shared by the all three business microservices, we already have the following line:
```
management.endpoints.web.exposure.include: "*"
```
Thus the `health` endpoint looks like to be exposed. Wasn't it?

Comment (MarianConstantinMarica) [reply spring-petclinic-vets-service/src/main/resources/application.yml:11]: Yes, it's correct i forgot to check on the config repo, I will update

Comment (arey) [reply spring-petclinic-vets-service/src/main/resources/application.yml]: Ok @MarianConstantinMarica 
For information, all Spring configuration have to be centralized into the `spring-petclinic-microservices-config` repository.

File: spring-petclinic-api-gateway/src/main/resources/application.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ spring:
   config:
     import: optional:configserver:${CONFIG_SERVER_URL:http://localhost:8888/}
   cloud:
+    loadbalancer:
+      configurations: health-check
```
Comment (arey) [inline spring-petclinic-api-gateway/src/main/resources/application.yml:8]: **question**: did you test this `health-check` confituration? and how? Did you have some logs?

File: spring-petclinic-vets-service/src/main/resources/application.yml [hunk 3]
```diff
@@ -8,6 +8,7 @@ spring:
   profiles:
     active: production
 
+
```
Comment (arey) [inline spring-petclinic-vets-service/src/main/resources/application.yml:11]: **question**: this pull request seems to be empty. Isn't it?

File: spring-petclinic-api-gateway/src/main/resources/application.yml (hunk 2 above)
Comment (arey) [reply spring-petclinic-api-gateway/src/main/resources/application.yml]: @MarianConstantinMarica  Did you have any logs?

**Pull request closed without merging**